modules/seqexec/server/src/main/scala/seqexec/server/ExecutionQueue.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server
+
+import cats.Eq
+import cats.implicits._
+import gem.Observation
+import monocle.macros.Lenses
+import seqexec.model.SequencesBatch.CommandState
+
+@Lenses
+final case class ExecutionQueue(name: String, cmdState: CommandState, queue: List[Observation.Id])
```

Comment:
Why not `QueueId` here?

Reply:
It would be redundant. `QueueId` is the key to the map of `ExecutionQueue`s